### DISCUSSION TITLE
Select Multiple Insight Panels

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#16541] [Feature Requests]
It would be ideal if you could click/drag to select multiple panels... and then drag them around as a group. It can be quite difficult to move them around now.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
